docs/SDK/Explanation/authentication/sessionSigs/authMethods/email-sms.md

```diff
@@ -0,0 +1,112 @@
+
+If you are using Lit Relay Server, you will need to request an API key [here](https://forms.gle/RNZYtGYTY9BcD9MEA).
+
+:::
+
+### Minting via Contract
+
+An alternative to minting the PKP NFT via the Lit Relay Server is to send a transaction to the smart contract yourself. You can reference the following example data that is passed to the `mintNextAndAddAuthMethods` method of the `PKPHelper` smart contract:
+
+- `keyType` is `2`
+- `permittedAuthMethodTypes` is `[7]`
+- `permittedAuthMethodIds` is an array with 1 element being the user's email or phone number.
+- `permittedAuthMethodScopes` is an array with 1 zero-initialized element, e.g. `[[ethers.BigNumber.from("0")]]`
+- `addPkpEthAddressAsPermittedAddress` is `true`
+- `sendPkpToItself` is `true`
+
+### Authetnicating to Fetch PKP information
+
+```javascript
+const authClient = new LitAuthClient({
+    litRelayConfig: {
+        relayApiKey: 'your-api-key',
+    }
+});
+
+// starting a validation session
+let session = authClient.initProvider(ProviderType.Otp,{
+            userId: 'user email or phone number'
+});
+
+let status = await session.sendOtpCode();
+let authMethod = await session.authenticate({
+    code: "user entered otp code"
+});
+const txHash = await session.fetchPKPThroughRelayer(authMethod);
+```
+:::note 
+If the user is using a phone number, the country code must be provided.
+:::
+
+Below is an example of an authentication method from successful authentication
+```javascript
+{
+    "accessToken": "eyJhbGciOiJzZWNwMjU2azEiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJMSVQtUHJvdG9jb2wiLCJzdWIiOiJMSVQtT1RQIiwiaWF0IjoxNjg0ODc1NTE0NDkxLCJleHAiOjE2ODQ4NzczMTQ0OTEsIm9yZ0lkIjoiTElUIiwicm9sZSI6InVzZXIiLCJleHRyYURhdGEiOiIrMTIwMTQwNzIwNzN8MjAyMy0wNS0yM1QyMDo1ODozNC40OTE3ODU5NDUrMDA6MDAifQ.eyJyIjoiZTA0ZDAyNjhjN2ExMzhiNmZiNDJjYTk4NmIxY2I4MWM0N2QyMTc0MzZlOWNlYzc4NGUzNWEyOTZkZmY2YjA4NSIsInMiOiI0NTE5MTVkMDY5YTZhZGE5M2U0OGY3ODUwMGM0MWUzNmMwYzQ4Y2FlODYwMmYxYWM0Njc0MTQ1YTNiMmMyNDU4In0",
+    "authMethodType": 7
+}
+```
+
+:::note 
+ Tokens expire after 30 minutes, and must be validated for session signature signing within that period
+:::
+
+
+## Generating `SessionSigs`
+
+After successfully authenticating with an `AuthMethod`, you can generate `Session Signatures` using the provider's `getSessionSigs` method. The `getSessionSigs` method takes in an `AuthMethod` object, a PKP public key, and other session-specific arguments such as `resourceAbilityRequests` and returns a `SessionSig` object.
+
+```javascript
+// Get session signatures for the given PKP public key and auth method
+const sessionSigs = await provider.getSessionSigs({
+  authMethod: '<AuthMethod object returned from authenticate()>',
+  sessionSigsParams: {
+    chain: 'ethereum',
+    resourceAbilityRequests: [{
+      resource: litResource,
+      ability: LitAbility.AccessControlConditionDecryption
+    }],
+  },
+});
+```
+
+# Technical Details
```

Comment:
i think this section is only really required if we provide details that aren't entirely mentioned above (see https://developer.litprotocol.com/SDK/Explanation/authentication/sessionSigs/authMethods/webAuthn#technical-details as an example)